source/uwp/winui3/ObjectModelCsProjection/ObjectModelCsProjection.csproj

```diff
@@ -1,7 +1,8 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net6.0-windows10.0.17763.0</TargetFramework>
     <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
+    <WindowsSdkPackageVersion>10.0.18362.38</WindowsSdkPackageVersion>
```

Comment:
The OS version here should probably match the TFM OS version.  In this case being 17763.  Same for others.

Reply:
I don't think that matters in this case match, right? `TargetPlatformMinVersion` is set to `17763` and (iirc) if we build successfully, we're not using any of the >=17763 APIs anyway :)
but I agree, it might be better to do it anyway.

Reply:
When I tried changing it to 10.0.17763.38 I got errors:
```
warning MSB3277: Found conflicts between different versions of "Microsoft.Windows.SDK.NET" that could not be resolved.
warning MSB3277: There was a conflict between "Microsoft.Windows.SDK.NET, Version=10.0.17763.38, Culture=neutral, PublicKeyToken=31bf3856ad364e35" and "Microsoft.Windows.SDK.NET, Version=10.0.18362.24, Culture=neutral, PublicKeyToken=31bf3856ad364e35".
warning MSB3277:     "Microsoft.Windows.SDK.NET, Version=10.0.17763.38, Culture=neutral, PublicKeyToken=31bf3856ad364e35" was chosen because it was primary and "Microsoft.Windows.SDK.NET, Version=10.0.18362.24, Culture=neutral, PublicKeyToken=31bf3856ad364e35" was not.

error NETSDK1148: A referenced assembly was compiled using a newer version of Microsoft.Windows.SDK.NET.dll. Please update to a newer .NET SDK in order to reference this assembly.
```

Reply:
does this happen when you set `WindowsSdkPackageVersion` to `10.0.17763.38` in **all** the project simultaneously?

Reply:
You are right that with respect to usage of APIs, `TargetPlatformMinVersion` will ensure you are using only the compatible ones.  But the projections for the different OS versions do have different assembly versions which is kind of seen above in the error.  This means if you have your projections shipping as compatible with 17763 TFM but use the 18362 Windows SDK projection package, and there was someone targeting 17763 that consumed your package, they will see the error similar to above.  Then they would need to move their TFM up to 18362.

@krschau which project did you see that error in?

Reply:
@beervoley I did make the change in all the projects simultaneously
@manodasanW I saw the error in the AdaptiveCardVisualizer project. If I keep only that one up on version 10.0.18362.38 then everything builds -- since that's just the test app, that's probably the way to go?

Reply:
The TFM for that project looks like to be targeting 22000, so that probably makes sense why you got that error..